src/radio.rs

```diff
@@ -68,7 +68,7 @@ pub struct BleRadio {
 
 impl BleRadio {
     // TODO: Use type-safe clock configuration to ensure that chip uses ext. crystal
-    pub fn new(radio: RADIO, ficr: &FICR, tx_buf: &'static mut PacketBuffer) -> Self {
+    pub fn new(radio: RADIO, _ficr: &FICR, tx_buf: &'static mut PacketBuffer) -> Self {
```

Comment:
If FICR is now unused, it would be better to just remove it instead of using `_ficr`. I've checked the datasheet and it didn't look like it contains anything else we might need.

Reply:
Done :)